Bento/Views/BentoReusableView.swift

```diff
@@ -20,6 +22,20 @@ extension BentoReusableView {
             containedView = nil
         }
     }
+
+    func willDisplayView() {
```

Comment:
As discussed I think we can move this methods to be part of `TableViewContainerCell` and `CollectionViewContainerCell` which hold both, `component` and `containedView`, and can invoke/forward this methods when they need to be aware of the lifecycle and that way we avoid exposing `AnyRenderable`.